_maps/RandomRuins/SpaceRuins/Immortus.dmm

```diff
@@ -0,0 +1,3406 @@
+	icon_state = "darkpurple";
+	dir = 4
+	},
+/area/ruin/powered/Immortus)
+"cB" = (
+/obj/structure/bed/roller,
+/obj/machinery/light/spooky/on,
+/turf/open/floor/plasteel/white,
+/area/ruin/powered/Immortus)
+"cC" = (
+/obj/structure/bed/roller,
+/turf/open/floor/plasteel/white,
+/area/ruin/powered/Immortus)
+"cD" = (
+/obj/structure/safe/floor,
+/obj/item/stack/sheet/mineral/diamond{
+	amount = 10
+	},
+/obj/item/stack/spacecash/c1000,
+/obj/item/stack/spacecash/c1000,
+/obj/item/stack/spacecash/c1000,
+/obj/item/stack/spacecash/c1000,
+/obj/item/stack/spacecash/c1000,
+/obj/item/stack/sheet/animalhide/gondola{
+	amount = 10
+	},
+/turf/open/floor/plasteel/whiteblue/side{
+	dir = 6
+	},
+/area/ruin/powered/Immortus)
+"cE" = (
+/obj/structure/closet/l3closet,
+/turf/open/floor/plasteel/white,
+/area/ruin/powered/Immortus)
+"cF" = (
+/obj/machinery/light/spooky/on,
+/turf/open/floor/plasteel/white,
+/area/ruin/powered/Immortus)
+"cG" = (
+/obj/structure/closet/coffin,
+/obj/item/device/gps/science{
+	gpstag = "Unauthorized Signal"
+	},
+/turf/open/floor/plasteel/white,
+/area/ruin/powered/Immortus)
+"cH" = (
+/obj/structure/table,
+/obj/item/reagent_containers/spray/cleaner{
+	pixel_x = 6
+	},
+/turf/open/floor/plasteel/white,
+/area/ruin/powered/Immortus)
+"cI" = (
+/obj/structure/tank_dispenser/oxygen,
+/turf/open/floor/plasteel/white,
+/area/ruin/powered/Immortus)
+"cJ" = (
+/obj/machinery/portable_atmospherics/canister/oxygen,
+/turf/open/floor/plasteel/white,
+/area/ruin/powered/Immortus)
+"cK" = (
+/obj/structure/table/glass,
+/obj/item/wallframe/camera,
+/obj/item/stack/cable_coil/cut,
+/obj/item/wirecutters{
+	pixel_y = 16
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"cL" = (
+/obj/structure/showcase/horrific_experiment,
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"cM" = (
+/obj/machinery/smoke_machine,
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"cN" = (
+/obj/machinery/atmospherics/components/unary/cryo_cell{
+	dir = 4
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"cO" = (
+/obj/machinery/atmospherics/components/unary/portables_connector/visible{
+	dir = 8
+	},
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"cP" = (
+/obj/structure/closet/body_bag,
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"cQ" = (
+/obj/structure/closet/radiation,
+/turf/open/floor/plasteel/darkpurple/side{
+	dir = 8
+	},
+/area/ruin/powered/Immortus)
+"cR" = (
+/obj/structure/closet/l3closet/security,
+/turf/open/floor/plasteel/darkpurple/side{
+	dir = 8
+	},
+/area/ruin/powered/Immortus)
+"cS" = (
+/obj/machinery/light/spooky{
+	dir = 8
+	},
+/turf/open/floor/plasteel/darkpurple/side{
+	dir = 8
+	},
+/area/ruin/powered/Immortus)
+"cT" = (
+/obj/structure/statue/haunted/angel,
+/turf/open/floor/plasteel/dark,
+/area/ruin/powered/Immortus)
+"cU" = (
+/obj/structure/table/reinforced,
+/obj/item/weldingtool/mini{
+	light_intensity = 0
```

Comment:
May I ask why?

Reply:
I figured out during testing that you could make yourself immune to the entire lighting puzzle with a lit welder. At the same time I wanted to make a welding tool available as part of a solution to the alarm room.